src/sagemaker/workflow/steps.py

```diff
@@ -320,6 +320,7 @@ def __init__(
         code: str = None,
         property_files: List[PropertyFile] = None,
         cache_config: CacheConfig = None,
+        kms_key=None,
```

Comment:
integ test?

Reply:
Let me remove this for the time being. It's an additional miss on the ProcessingStep as you can't specify a KMS key for the code uploaded by `_normalize_inputs`, but it's not related to the problem this PR is solving. I'll make a new PR for this.